packages/oruga/src/components/steps/tests/StepsExample.vue

```diff
@@ -0,0 +1,127 @@
+<script setup lang="ts" generic="T extends Record<string, unknown> | string">
+import { type PropType, ref } from "vue";
+import OSteps from "@/components/steps/Steps.vue";
+import OStepItem from "@/components/steps/StepItem.vue";
+import OButton from "@/components/button/Button.vue";
+
+defineProps({
+    isVertical: {
+        type: Boolean,
+        default: false,
+    },
+    isAnimated: {
+        type: Boolean,
+        default: false,
+    },
+    isRounded: {
+        type: Boolean,
+        default: false,
+    },
+    hasNavigation: {
+        type: Boolean,
+        default: false,
+    },
+    labelPosition: {
+        type: String as PropType<"bottom" | "left" | "right">,
+        default: "bottom",
+    },
+    isStepsClickable: {
+        type: Boolean,
+        default: false,
+    },
+    isProfileSuccess: {
+        type: Boolean,
+        default: false,
+    },
+    showSocial: {
+        type: Boolean,
+        default: false,
+    },
+    customNavigation: {
+        type: Boolean,
+        default: false,
+    },
+});
+
+const activeStep = ref<number>(1);
+const enableProfileActivateEvent = ref(false);
+
+const onProfileActivate = () => {
+    if (enableProfileActivateEvent.value) {
+        alert("Profile Activated");
```

Comment:
I think using alert here seems wrong. I don't know how it works with a virtual dom?

Reply:
I copied this from the docs. Happy to remove or replace. I think as long as there is any func here it should satisfy our testing needs